drafts/sustainable.md

```diff
@@ -0,0 +1,36 @@
+# Sustainable
+
+Remaking the financial system in a more fair and inclusive way will be a journey, and Decred and its community must operate on a long time horizon. To realize Decred as a superior long-term store of value, every effort and expenditure needs to serve its long term goals.
```

Comment:
I believe there should be a hyphen in "long-term". It's at the end of the last sentence.